sentry/src/test/java/io/sentry/metrics/MetricsApiTest.kt

```diff
@@ -206,7 +225,91 @@ class MetricsApiTest {
                 )
             ),
             anyOrNull(),
+            anyOrNull(),
             anyOrNull()
         )
     }
+
+    @Test
+    fun `local aggregator is provided to aggregator`() {
+        val aggregator = mock<IMetricsAggregator>()
+        val localMetricsAggregator = mock<LocalMetricsAggregator>()
+
+        val api = MetricsApi(object : IMetricsInterface {
```

Comment:
I would extract this into a `Fixture` class, and the `verify`s into some private methods